docs: Update Docs : Create session remote server URL

sapandang opened the pull request:
## As tested with the below configuration:
* Device platform : Android 
* PC OS : Windows
* Language : JAVA
Remote driver does not connect to **appium desktop server** without appending `/wd/hub` to the URL

## Proposed changes
Users are not able to connect to "http://127.0.0.1:4723" need to append `/wd/hub` to the url.
Adding the comment will enable to connect to  appium desktop server easily.
> So just added a comment in the example use for java only. 

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
None

sapandang committed:
Remote server URL

As tested with the below configuration:
Device platform : Android 
PC OS : Windows
Remote driver does not connect without appending `/wd/hub` to the URL

KazuCocoa commented on commands-yml/commands/session/create.yml:
```diff
@@ -18,7 +18,8 @@ example_usage:
 
         URL url = new URL("http://127.0.0.1:4723");
-
+        //platform- android : windows
```
Thanks! You can remove this line

sapandang replied:
If I am right, I should remove the comment and replace `URL url = new URL("http://127.0.0.1:4723");` with `//URL url = new URL("http://127.0.0.1:4723/wd/hub"); ` ?

KazuCocoa replied:
yea, it is also good

sapandang replied:
changes done.

sapandang committed:
Removing comment and changing remote url

removed the comment and replaced URL url = new URL("http://127.0.0.1:4723"); with //URL url = new URL("http://127.0.0.1:4723/wd/hub"); ?

KazuCocoa commented on commands-yml/commands/session/create.yml:
```diff
@@ -16,9 +16,9 @@ example_usage:
-
-        URL url = new URL("http://127.0.0.1:4723");
-
+       
```
blank spaces in L19, 20 and 21

sapandang replied:
L19 and 21 already had spaces.
as in http://appium.io/docs/en/commands/session/create/
In my opinion it should be left as it is since it increase the readability

KazuCocoa replied:
I meant below highlighted diff.
![image](https://user-images.githubusercontent.com/5511591/74798909-68bdb000-5312-11ea-89ef-45b8d3d26c2a.png)

I expected the diff was only `/wd/hub`

KazuCocoa commented:
thanks!
